src/libs/ReportUtils.js

```diff
@@ -310,19 +310,28 @@ function hasExpensifyEmails(emails) {
     return _.intersection(emails, CONST.EXPENSIFY_EMAILS).length > 0;
 }
 
+/**
+ * Returns an array after removing expensify emails
+ * @param {Array} emails
+ * @returns {Array}
+ */
+function removeExpensifyEmails(emails) {
+    return _.difference(emails, CONST.EXPENSIFY_EMAILS);
+}
+
 /**
  * Whether the time row should be shown for a report.
  * @param {Array<Object>} personalDetails
  * @param {Object} report
  * @return {Boolean}
  */
 function canShowReportRecipientLocalTime(personalDetails, report) {
-    const reportParticipants = lodashGet(report, 'participants', []);
-    const hasMultipleParticipants = reportParticipants.length > 1;
-    const reportRecipient = personalDetails[reportParticipants[0]];
+    const reportParticipants = _.without(lodashGet(report, 'participants', []), sessionEmail);
+    const participantsWithoutExpensifyEmails = removeExpensifyEmails(reportParticipants);
```

Comment:
Honestly, since this is on a single line method I'd just use `_.difference(emails, CONST.EXPENSIFY_EMAILS);` here instead of creating a whole new method for it

Reply:
Hmm, I think we should keep it as we are using it at two places and this function can be useful in the future to deal with something like this. What you think?

Reply:
I personally don't really see the value since it doesn't really add any readability since what the __difference is doing is pretty clear. You can keep it if you want, this is not a blocker

Reply:
> Can we also see a video for:
> 
> Chat with concierge (No timezone should be shown) Chat with 1:1 (Timezone should be shown) Chat with another user and concierge ( Timezone should be shown)

Sure, will attach the video after removing the function.